backend/controller/bookController.js

```diff
@@ -31,23 +31,66 @@ export const getAllBooks = async (req, res) => {
 // @desc		Get books by author name or genre title or ISBN
 // @route		GET	/books/search/:searchQuery
 
+// export const getBooksBySearchQuery = async (req, res) => {
+// 	try {
+// 		const searchQuery = req.params.searchQuery;
+// 		const booksBySearchQuery = await Book.find({
+// 			$or: [
+// 				{ author: { $regex: searchQuery, $options: 'i' } },
+// 				{ title: { $regex: searchQuery, $options: 'i' } },
+// 				{ ISBN: searchQuery },
+// 				{ genre: searchQuery },
```

Comment:
We can optimize this search by specifying the field rather than looking for all of them. What do people think?